### ISSUE TITLE
Price in "Materials" tab and Spool price always show euros.

### ISSUE THREAD

**Issue by oleost** [CNCKitchen/3D-Printing-costs#1]
Hi.

I downloaded the sheet and imported it to google sheet.

I changed money unit to "NOK" but it still shows euros.

![image](https://user-images.githubusercontent.com/5670710/31320467-571c71ba-ac75-11e7-893f-1abe8a141787.png)

**Comment by johnbiggs** [comment]
Yes, it doesn't seem easy to fix this so I recommend removing the unit from this column.

**Comment by nsidious9** [comment]
it is a super easy fix.  select the letter of the column and then select the denomination where the $ is
